presentation/src/main/java/me/androidbox/presentation/task/screen/TaskDetailScreen.kt

```diff
@@ -54,6 +58,24 @@ fun TaskDetailScreen(
     modifier: Modifier = Modifier
 ) {
     val calendarStateTimeDate = rememberUseCaseState()
+    val snackbarHostState = remember {
+        SnackbarHostState()
+    }
+    val coroutineScope = rememberCoroutineScope()
+    var snackbarJob: Job? = null
+
+    LaunchedEffect(key1 = taskDetailScreenState.showSnackBar) {
+        if(taskDetailScreenState.showSnackBar) {
+            snackbarJob?.cancel()
```

Comment:
I added a cancel here, as if the user clicks save multiple times, I just want to dismiss the current snackbar and then show it again. What do you think?

Reply:
That's what `LaunchedEffect` already does. Every time the passed key changes, the coroutine is cancelled and relaunched

Reply:
@philipplackner Will remove the snackbarJob.cancel()